docs/hub/bertopic.md

```diff
@@ -0,0 +1,78 @@
+# Using BERTopic at Hugging Face
+
+[BERTopic](https://github.com/MaartenGr/BERTopic) BERTopic is a topic modeling framework that leverages 🤗 transformers and c-TF-IDF to create dense clusters allowing for easily interpretable topics whilst keeping important words in the topic descriptions. 
```

Comment:
```suggestion
[BERTopic](https://github.com/MaartenGr/BERTopic) is a topic modeling framework that leverages 🤗 transformers and c-TF-IDF to create dense clusters allowing for easily interpretable topics whilst keeping important words in the topic descriptions. 
```

Reply:
`BERTopic` would still be repeated here.

Reply:
Sorry, overlooked your comments! I just pushed the changes for this piece of feedback as well as the one below.